### ISSUE TITLE
Static files placed in incorrect location when `.` used as source

### ISSUE THREAD

**Issue by leesdolphin** [cobalt-org/cobalt.rs#183]
When `.cobalt.yml`'s `source` variable is equal to `.`, all static files are copied into the root of the output folder with their name being their extension.

To reproduce:
```
mkdir cobalt-test
cd cobalt-test
cobalt new
touch myimg.png
cobalt build
```
`build/myimg.png` should now exist; however it does not, and `build/png` exists.

Version information:
```
$ cobalt --version
Cobalt 0.5.0
```

**Comment by nott** [comment]
@leesdolphin could you please mention cobalt version you are using?
I think the issue has been fixed in #175

**Comment by leesdolphin** [comment]
@nott, I am using Cobalt 0.5.0. Sorry about not including it, accidentally submitted instead of adding a new line

**Comment by leesdolphin** [comment]
I'm just going to try the latest master version now to confirm if that fixes it.

**Comment by nott** [comment]
@leesdolphin #175 went into 0.5.0: https://github.com/cobalt-org/cobalt.rs/commits/v0.5.0

so that's probably another issue

**Comment by leesdolphin** [comment]
Ahh, yeah, I just checked against master too. Luckily it's easily remedied by the user by changing the `source` to `./`.

**Comment by nott** [comment]
@leesdolphin thank you for working on this!

**Comment by tak1n** [comment]
Should be fixed by #191
